google/cloud/spanner/keys.cc

```diff
@@ -14,12 +14,24 @@
 
 #include "google/cloud/spanner/keys.h"
 #include "google/cloud/spanner/value.h"
+#include <google/protobuf/util/message_differencer.h>
 #include <google/spanner/v1/keys.pb.h>
 
 namespace google {
 namespace cloud {
 namespace spanner {
 inline namespace SPANNER_CLIENT_NS {
+
+bool operator==(KeySet const& lhs, KeySet const& rhs) {
+  //    std::string delta;
+  google::protobuf::util::MessageDifferencer differencer;
+  //    differencer.ReportDifferencesToString(&delta);
```

Comment:
I'd remove these two commented lines since their not being used and we don't really have a place to log the `delta` here anyway.

Reply:
Forgot I had left all that cruft in here. That's what I get for being in a hurry.